src/components/BTextarea/BTextarea.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div :class="classes">
+    <label :for="id">
+      {{ label }}<span v-if="required" aria-label="Required">*</span>
+    </label>
+    <textarea
+      v-model="inputValue"
+      v-bind="inputAttributes"
+    />
+    <div class="b-input-group--error">
+      <span></span>
+    </div>
+  </div> 
+</template>
+
+<script>
+export default {
+  name: 'b-textarea',
+  props: {
+    id: {
+      type: String,
+      required: true
+    },
+    name: {
+      type: String,
+      required: true
+    },
+    cols: {
+      type: Number,
+      default: 30
+    },
+    rows: {
+      type: Number,
+      default: 5
+    },
+    placeholder: String,
+    label: {
+      type: String,
+      required: true
+    },
+    required: {
```

Comment:
Are you going to add v-validate validation to this?

Reply:
As it wasn't a part of the story, I wasn't going to. And we hadn't decided on actually using it for the whole project.